Header header 
float64 Lat
float64 Lon
float64 UTM_X
float64 UTM_Y
float64 Angle
float64 map_x
float64 map_y

float64 raw_UTM_X
float64 raw_UTM_Y
float64 raw_Angle
float64 raw_map_x
float64 raw_map_y

float64 mqtt_angle

float64 Vel
int8 Quality
int8 HeartBeat


